app/models/user.rb

```diff
@@ -2,7 +2,8 @@ class User < ApplicationRecord
   PROPERTIES = %i(name email address phone password
                   password_confirmation remember_me).freeze
   devise :database_authenticatable, :registerable, :validatable,
-         :confirmable, :recoverable, :rememberable
+         :confirmable, :recoverable, :rememberable,
+         :trackable, :timeoutable, :lockable
```

Comment:
Tác dụng của từng module này là gì em nhỉ

Reply:
- confirmable: có tác dụng gửi mail xác thực tài khoản,
- recoverable: dùng để khôi phục lại mật khẩu
- rememberable: ghi nhớ người dùng đăng nhập bằng cách lưu trên cookie trình duyệt
- trackable: dùng để theo dõi user, địa chỉ ip, thời gian đăng nhập, số lần đăng nhập
- lockable: dùng để bảo mật tài khoản, khi mà user đăng nhập nhiều lần mình có thể khóa tài khoản lại, và có thể gửi mail để mở lại tài khoản.
- timeoutable: có tác dụng khi user không làm việc trong 1 khoản thời gian thì sẽ logout ra

Reply:
@anlvq-1058 anh cho em hỏi cái timeoutable, giờ em muốn chỉ ở trang admin, sau khi user không sử dụng quá 5p thì sẽ logout ra, thì mình có cách xử lý nào ko ạ, hiện tại thì em làm ở toàn web luôn ạ

Reply:
@tuongbx-1220 anh cũng không biết cách nào cả haha